src/dao/BaseDAO.js

```diff
@@ -2,26 +2,23 @@
 import type {
   DAOConfig,
   DAOTranslator,
-  EntityID,
   EntityName,
   QueryFilter,
   QueryOptions,
   RequestMethod,
 } from '../index';
 
-import nullthrows from 'nullthrows';
 import oHandler from 'odata';
-import LoadObject from '../load_object/LoadObject';
+import LoadObject from '../LoadObject';
 import { FILTER_FUNCTION_OPERATORS } from '../constants';
-import { createFilter } from '../filters';
 
 const ID_REG_EXP = /\bid\b/;
 
 class BaseDAO<TEntity, TEntityMutator> {
   static _organizationID: ?string = null;
   __config: DAOConfig<TEntity, TEntityMutator>;
-  _entityLoaderByID: Map<EntityID, LoadObject<TEntity>> = new Map();
-  _entityIDsLoaderByQuery: Map<string, LoadObject<Array<EntityID>>> = new Map();
+  _entityLoaderByID: Map<string, LoadObject<TEntity>> = new Map();
```

Comment:
Even though `EntityID` is just a string, I still want to use it as a type.

It makes things more readable and it's easier to understand that you're working with an ID